pkgs/by-name/mi/microfetch/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "microfetch";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "NotAShelf";
+    repo = "microfetch";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-iBfnui7xrx31euYnQxoJf1xXFYFiJnDMadLRAiRCN04=";
+  };
+
+  cargoHash = "sha256-J+S6XIMUcWj4jXABQinEDx6dRG5Byc2UxJoKj2y1tQU=";
+
+  meta = {
+    description = "Microscopic fetch script in Rust, for NixOS systems";
+    homepage = "https://github.com/NotAShelf/microfetch";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ nydragon ];
+    mainProgram = "microfetch";
```

```suggestion
    mainProgram = "microfetch";
    platforms = lib.platforms.linux;
```
https://github.com/NotAShelf/microfetch/blob/927f6077b4b78cd112bc6932aab5287edc413ce8/flake.nix#L9

---

Fixed :+1: